tools/tertiary-analysis/data/convert-ftype.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<tool id="convert-ftype" name="RDS to AnnData converstion" version="v0.0.1+galaxy0">
+  <requirements>
+    <requirement type="package" version="2.3.1">r-seurat</requirement>
+  </requirements>
+  <description>using seurat</description>
+  <command detect_errors="exit_code"><![CDATA[
+ln -s '${input_rds}' input.rds &&
+cp ${script_file} rds2anndata.R &&
```

Comment:
why not `Rscript ${script_file}` directly and avoid the cp?

Reply:
Yes, that should work.